pro/src/components/pages/Offers/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -510,6 +534,8 @@ const OfferForm = ({
             }
             // front should check categoryId but do not send to backend
             delete submittedValues.categoryId
+            // remove withdrawalType waiting for the back to be ready
```

Comment:
peut etre tagé en todo
```suggestion
            // TODO: remove withdrawalType waiting api merge of PC-.... (ou api merge of <pr hash>)
```